### PR TITLE
Updates to TorchServe page on pytorch.org

### PR DESCRIPTION
**Pull request by agunapal**
## Description

Update TorchServe's documentation in https://pytorch.org/serve/

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Feature/Issue validation/testing

NA

## Checklist:

- [ ] Did you have fun?
- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,8 +1,24 @@
 # TorchServe
 
-TorchServe is a performant, flexible and easy to use tool for serving PyTorch eager mode and torschripted models.
+TorchServe is a performant, flexible and easy to use tool for serving PyTorch eager mode and torchscripted models.
```
Comment (msaroufim) [inline docs/index.md:3]: I don't think torchscript should be in in our one liner anymore

**Review by msaroufim — APPROVED** [review]
One minor nit

File: docs/index.md (hunk 1 above)
Comment (min-jean-cho) [reply docs/index.md:3]: Should we mention compiled mode in replacement of torchscript?

File: docs/index.md [hunk 2]
```diff
@@ -50,3 +71,28 @@ TorchServe is a performant, flexible and easy to use tool for serving PyTorch ea
 * [mlflow-torchserve](https://github.com/mlflow/mlflow-torchserve) - Deploy mlflow pipeline models into TorchServe
 * [Kubeflow pipelines](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/pytorch-samples) - Kubeflow pipelines and Google Vertex AI Managed pipelines
 * [NVIDIA MPS](mps.md) - Use NVIDIA MPS to optimize multi-worker deployment on a single GPU
+
+## 📰 News
+* [Torchserve Performance Tuning, Animated Drawings Case-Study](https://pytorch.org/blog/torchserve-performance-tuning/)
+* [Walmart Search: Serving Models at a Scale on TorchServe](https://medium.com/walmartglobaltech/search-model-serving-using-pytorch-and-torchserve-6caf9d1c5f4d)
+* [🎥 Scaling inference on CPU with TorchServe](https://www.youtube.com/watch?v=066_Jd6cwZg)
+* [🎥 TorchServe C++ backend](https://www.youtube.com/watch?v=OSmGGDpaesc)
+* [Grokking Intel CPU PyTorch performance from first principles: a TorchServe case study](https://pytorch.org/tutorials/intermediate/torchserve_with_ipex.html)
```
Comment (min-jean-cho) [inline docs/index.md:80]: Could we please also add link to Part 2? https://pytorch.org/tutorials/intermediate/torchserve_with_ipex_2.html?highlight=grokking%20pytorch%20intel%20cpu%20performance%20from%20first%20principles%20part

**Review by min-jean-cho — APPROVED** [review]
left some minor comments, otherwise LGTM

File: docs/index.md (hunk 1 above)
Comment (agunapal) [reply docs/index.md:3]: Thanks. I'm going to just modify it to "serving PyTorch models in production"

File: docs/index.md (hunk 2 above)
Comment (agunapal) [reply docs/index.md:80]: Done. Also added in README

**Pull request merged**